src/core.js

```diff
@@ -381,6 +381,12 @@ $.extend( $.validator, {
 			} );
 
 			function delegate( event ) {
+
+				// Set form expando on contenteditable
+				if ( this.hasAttribute( "contenteditable" ) ) {
```

Comment:
Thanks for that, you're absolutely right, this seems to have been lost along the process. 
Added it back and should be good to go.